src/Discord.Net.WebSocket/Entities/Users/SocketWebhookUser.cs

```diff
@@ -24,6 +24,13 @@ public class SocketWebhookUser : SocketUser, IWebhookUser
         public override ushort DiscriminatorValue { get; internal set; }
         /// <inheritdoc />
         public override string AvatarId { get; internal set; }
+
+        /// <inheritdoc />
+        public override string BannerId { get; internal set; }
+
+        /// <inheritdoc />
+        public override Color? AccentColor { get; internal set; }
```

Comment:
webhooks cannot have banner/accent colors, please make it throw a NotSupportedException

Reply:
its just `null` 🤔

Reply:
Resolved in abcb4e7a9a524d79d54ff980629ffebb36ae54f7